nanoFramework.Json.Test/JsonUnitTests.cs

```diff
@@ -1384,6 +1384,34 @@ public void DeserializeObjectWithStringContainingNonAsciiChars()
             Assert.Equal(input.Value, result.Value);
         }
 
+        [TestMethod]
+        public void DeserializeObjectWithJsonIgnoreAttribute()
+        {
+            OutputHelper.WriteLine("Starting JsonIgnore Test...");
+            Json.Configuration.Settings.UseIgnoreAttribute = true;
+            OutputHelper.WriteLine("UseIgnoreAttribute enabled.");
+
+            var testObject = JsonIgnoreTestClass.CreateTestClass();
+            var jsonString = JsonSerializer.SerializeObject(testObject);
+            OutputHelper.WriteLine("After serialize.");
+            JsonIgnoreTestClass dserResult = JsonConvert.DeserializeObject(jsonString, typeof(JsonIgnoreTestClass)) as JsonIgnoreTestClass;
+            OutputHelper.WriteLine("After deserialize.");
+
+            //test serialize and deserialize
```

Comment:
we use to add a space between the // comment and a start with uppercase. IT's minimal ;-) Also applies elsewhere

Reply:
To clarify, comments that are NOT for intellisense don't need to have the period at the end, but should have a leading space and start with uppercase? 
Whereas, intellisense comments should be formatted as full sentences?

Reply:
Yes and yes!